l10n.md

```diff
@@ -0,0 +1,48 @@
+# Translating Ladybird
+## gettext
+Ladybird is translated with the help of [gettext](https://www.gnu.org/software/gettext/).
+
+Add a new translation:
+
+### 1. Add a new po file
+
+You can find current translations in https://github.com/bugaevc/ladybird-gtk4/tree/master/po, copy one of the files and translate all the strings `msgstr` **[what is the command the user should run instead of copying one of the current files?]**
+
+You also have to change some meta information:
+```
+# Swedish translation for Ladybird.
+"Language: sv\n"
+"Plural-Forms: nplurals=2; plural=n != 1;\n"
+```
+Read more about [Plural-Forms](https://www.gnu.org/software/gettext/manual/html_node/Plural-forms.html)
+
+### 2. CMakeLists.txt
+
+Add your .po-file to [/po/CMakeLists.txt](https://github.com/bugaevc/ladybird-gtk4/blob/master/po/CMakeLists.txt)
+```
+    set(linguas
+        eo.po
+        ru.po
+        sv.po
+    )
+```
+### 3. org.serenityos.Ladybird-gtk4.desktop
+Translate the strings in [org.serenityos.Ladybird-gtk4.desktop](https://github.com/bugaevc/ladybird-gtk4/blob/master/org.serenityos.Ladybird-gtk4.desktop)
```

Comment:
Link to https://specifications.freedesktop.org/desktop-entry-spec/desktop-entry-spec-latest.html#localized-keys here somewhere